src/index.d.ts

```diff
@@ -184,15 +184,16 @@ export class DiffPatcher {
     unpatch: (right: any, delta: Delta) => any;
 }
 
-export const create: (options?: any) => DiffPatcher
+export const create: (options?: any) => DiffPatcher;
 
 export const formatters: {
   annotated: Formatter;
   console: Formatter;
   html: HtmlFormatter;
+  jsonpatch: Formatter;
```

Comment:
This actually probably isn't quite right because `Formatter.format` currently returns a `string` and according to https://github.com/benjamine/jsondiffpatch/pull/316 it returns an array of operations. We'll get this fixed up in a future PR.